addons/binding/org.openhab.binding.amazondashbutton/README.md

```diff
@@ -112,17 +114,19 @@ You can ignore these devices in your Inbox.
 
 ### Amazon Dash Button
 
-*   `macAddress` - The MAC address of the Amazon Dash Button.
-*   `pcapNetworkInterfaceName` - The network interface which receives the packets of the Amazon Dash Button.
-*   `packetInterval` - Often a single button press is recognized multiple times.
-You can specify how long any further detected button pressed should be ignored after one click was processed.
-The parameter is optional and 5000ms by default.
+-   `macAddress` - The MAC address of the Amazon Dash Button.
+
+-   `pcapNetworkInterfaceName` - The network interface which receives the packets of the Amazon Dash Button.
+
+-   `packetInterval` - Often a single button press is recognized multiple times.
+    You can specify how long any further detected button pressed should be ignored after one click was processed.
+    The parameter is optional and 5000ms by default.
```

Comment:
Consider changing to, "This parameter is optional, and is set to 5000ms by default."